std/algorithm/comparison.d

```diff
@@ -1088,6 +1088,14 @@ the minimal amount of edit operations necessary to transform $(D s)
 into $(D t).  Performs $(BIGOH s.length * t.length) evaluations of $(D
 equals) and occupies $(BIGOH s.length * t.length) storage.
 
+Params:
+    equals = The binary predicate to compare the elements of the two ranges.
+    s = The original range.
+    t = The transformation target
+
+Returns:
+    The minimal amount of edits to transform s into t.
```

Comment:
Nitpick: better to use "number of edits" instead of "amount of edits", which sounds ambiguous.